controllers/helm.go

```diff
@@ -109,6 +87,49 @@ func mergeHelmConfigs(meta *ectypes.ReleaseMetadata, in *v1beta1.Installation) *
 	return combinedConfigs
 }
 
+// update the 'admin-console' and 'embedded-cluster-operator' charts to add cluster ID, binary name, and airgap status
+func updateInfraChartsFromInstall(in *v1beta1.Installation, charts k0sv1beta1.ChartsSettings) k0sv1beta1.ChartsSettings {
+	if in == nil {
+		return charts
+	}
+
+	for i, chart := range charts {
+		if chart.Name == "admin-console" {
+			// admin-console has "embeddedClusterID" and "isAirgap" as dynamic values
+			newVals, err := setHelmValue(chart.Values, "embeddedClusterID", in.Spec.ClusterID)
+			if err != nil {
+				fmt.Printf("failed to set embeddedClusterID for %s: %v", chart.Name, err)
```

Comment:
can we replace these with the `log` as in `installation_controller.go` ?

Reply:
whoops I should have removed that entirely

Reply:
yep yep, will do